.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -20,6 +20,32 @@ This issue captures the state of the OpenSearch release, its assignee (Release M
 
 Please refer to the following link for the release version dates: [Release Schedule and Maintenance Policy](https://opensearch.org/releases.html).
 
+### [Entrance Criteria](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#entrance-criteria-to-start-release-window)
+Criteria | Status | Description  | Comments
+-- | -- | -- | --
+Documentation draft PRs are up and in tech review for all component changes | :red_circle: |   |
+Sanity testing is done for all components | :red_circle: |   |
+Code coverage has not decreased (all new code has tests) | :red_circle: |   |
+Release notes are ready and available for all components | :red_circle: |   |
+Roadmap is up-to-date (information is available to create release highlights) | :red_circle: |   |
+Release ticket is cut, and there's a forum post announcing the start of the window | :red_circle: |   |
+[Any necessary security reviews](##Security-Reviews) are complete | :red_circle: |   |
+
+### OpenSearch {{ env.VERSION }} [exit criteria](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#exit-criteria-to-close-release-window) status:
+Criteria | Status | Description  | Comments
+-- | -- | -- | --
+Performance tests are run, results are posted to the release ticket and there no unexpected regressions | :red_circle: |   |
+Documentation has been fully reviewed and   signed off by the documentation community. | :red_circle: |   |
+All integration tests are passing |  :red_circle: |   |
+Release blog is ready | :red_circle: |   |
+
+### OpenSearch-Dashboards {{ env.VERSION }} [exit criteria](https://github.com/opensearch-project/.github/blob/main/RELEASING.md#exit-criteria-to-close-release-window) status:
+Criteria | Status | Description  | Comments
+-- | -- | -- | --
+Documentation has been fully reviewed and   signed off by the documentation community | :red_circle: |   |
+All integration tests are passing | :red_circle: |   |
+Release blog is ready | :red_circle: |   |
+
```

Comment:
Can we add something about all med and high CVEs older than 60 days have been addressed?

Reply:
I think we need to add that to the releasing.md as well.

Reply:
Should this be in main issue as well as child issues? (Basically build repo issue and all components issues as well?)

Reply:
Added to the main issue for now.

Reply:
I think it makes sense to push it out to all the repos, but WDYT?

Reply:
I believe @bbarani raised a point that if some criteria changes, how do we keep track of all places to change it. Its like how we update dates. We mention them in main issues and refer the child components to go look there. 
If the criteria is not gonna be that dynamic then I am okay to add them in child issues too for now.